extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -289,6 +290,21 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
     /** @private {number} */
     this.ifi_ = 0;
+
+    /** @private {boolean} */
+    this.isFluid_ = false;
+
+    /** @private {?string} */
+    this.fluidImpressionUrl_ = null;
+
+    /** @private {boolean} */
+    this.fluidImpressionFired_ = false;
```

Comment:
Why not just set fluidImpressionUrl_ to null on success?

Reply:
Done.